gdx/src/com/badlogic/gdx/graphics/g3d/utils/shapebuilders/RenderableShapeBuilder.java

```diff
@@ -123,11 +124,11 @@ public static void buildNormals (MeshPartBuilder builder, Renderable renderable,
 
 		if (mesh.getNumIndices() > 0) {
 			// Get min vertice to max vertice in indices array
-			ensureIndicesCapacity(mesh.getNumIndices());
```

Comment:
Why did you remove `ensureIndicesCapacity`?

Reply:
@dar-dev

Reply:
it was originally referring to a static storage.

Reply:
caching a float array here is a bit of overkill imho. Also I did not find any reference to RenderableShapeBuilder in LIBGDX. Am I wrong ?

Reply:
Double checked it. No way to optimize unless we change the visibility of some unreferenced methods (and we want to keep the libgdx api). But we can remove the old ensureIndicesCapacity().